### DISCUSSION TITLE
Outputting a .vtk file for a simple example not working

### DISCUSSION THREAD

**Discussion by kurtenkera** [idaholab/moose#27277] [Q&A General]
I have an input file (see below) titled `pressure_diffusion.i`. I'm trying to output .vtk file types (as I'm interested in then converting the .vtk file to a .msh file for a project using `meshio`). Currently, I do this by setting `vtk = true` in the `[Outputs]` section within the input file. However, when I execute the input file, I don't get a .vtk file. Instead I get four files titled: `pressure_diffusion_out_001.pvtu`, `pressure_diffusion_out_001_0.vtu`, `pressure_diffusion_out_000.pvtu` and `pressure_diffusion_out_000_0.vtu`. This link (link: https://mooseframework.inl.gov/syntax/Outputs/index.html) says that VTK output types require `--enable-vtk` "when building libMesh", but I don't know what this means? 

How can I output a .vtk file type which I can then convert to a .msh file using some other intermediate file conversion package? 

```
[Mesh]
  type = GeneratedMesh # Can generate simple lines, rectangles and rectangular prisms
  dim = 2 # Dimension of the mesh
  nx = 100 # Number of elements in the x direction
  ny = 10 # Number of elements in the y direction
  xmax = 0.304 # Length of test chamber
  ymax = 0.0257 # Test chamber radius
[]

[Problem]
  type = FEProblem # This is the "normal" type of Finite Element Problem in MOOSE
  coord_type = RZ # Axisymmetric RZ
  rz_coord_axis = X # Which axis the symmetry is around
[]

[Variables]
  [pressure]
    # Adds a Linear Lagrange variable by default
  []
[]

[Kernels]
  [diffusion]
    type = ADDiffusion # Laplacian operator
    variable = pressure # Operate on the "pressure" variable from above
  []
[]

[BCs]
  [inlet]
    type = ADDirichletBC # Simple u=value BC
    variable = pressure # Variable to be set
    boundary = left # Name of a sideset in the mesh
    value = 4000 # (Pa) From Figure 2 from paper. First data point for 1mm spheres.
  []
  [outlet]
    type = ADDirichletBC
    variable = pressure
    boundary = right
    value = 0 # (Pa) Gives the correct pressure drop from Figure 2 for 1mm spheres
  []
[]

[Executioner]
  type = Steady # Steady state problem
  solve_type = NEWTON # Perform a Newton solve

  # Set PETSc parameters to optimize solver efficiency
  petsc_options_iname = '-pc_type -pc_hypre_type' # PETSc option pairs with values below
  petsc_options_value = ' hypre    boomeramg'
[]

[Outputs]
  vtk = true
[]
```

**Comment by GiudGiud** [comment]
Hello

There are a few VTK-like formats. vtk, vtu, pvtu etc.
I think the VTK documentation describes them better than I can. We output pvtu to be able to perform parallel IO.
Hopefully meshio can convert .pvtus. If not the paraview can be used to convert to something meshio can read

Guillaume
